tests/onnx/data/reference_graphs/quantization/synthetic/multi_input_output_model.dot

```diff
@@ -0,0 +1,24 @@
+strict digraph  {
```

Comment:
Where is the original model graph?

Reply:
The original is already added. The quantization of that model was skipped before. I removed skipping